### ISSUE TITLE
Ctl + Shift Selection

### ISSUE THREAD

**Issue by bgpalmer** [glin/reactable#325]
Hi,

I apologize if this has been asked already or if I'm missing it in the documentation - but it would be nice to be able to ctrl + shift selection of multiple rows in shiny. Currently, I allow users to select multiple rows with `multiple=TRUE` but it does not seem to allow ctl + shift multiple selection. Could this functionality be added to the package? 

Thanks
Brian

**Comment by glin** [comment]
I agree, and it's something I need myself so I've been trying to find time to do it recently. It'll probably use the Shift key only though, as I believe that's the usual convention across apps.

**Comment by bgpalmer** [comment]
You're right, it should just be the shift key. I've updated the title
